packages/contracts-bedrock/src/deploy-utils.ts

```diff
@@ -276,3 +276,12 @@ export const getDeploymentAddress = async (
   const deployment = await hre.deployments.get(name)
   return deployment.address
 }
+
+export const jsonifyTransaction = (tx: ethers.PopulatedTransaction): string => {
```

Comment:
do we want to stringify in a more human readable way, ie `JSON.stringify({}, nulll, 2)`?

Reply:
Sure, I can do this